src/hb-ft.cc

```diff
@@ -675,6 +746,16 @@ hb_ft_font_create (FT_Face           ft_face,
   return font;
 }
 
+/**
+ * hb_ft_font_has_changed:
+ * @font: #hb_font_t to work upon
+ *
+ * Refreshes the state of @font when the underlying FT_Face has changed.
```

Comment:
BTW (and this does not have to be addressed in this PR), can we add a markup to FreeType objects and functions and make them link to FreeType documentation?

Reply:
The markup part is easy. Linking less so. But it is a good idea. Maybe just opening an issue about it would be the best.

Reply:
https://github.com/harfbuzz/harfbuzz/issues/2347